### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by 3TUSK**
~~Don't merge this yet. Feedback wanted. @TartaricAcid~~

Edit: Ready to go.

### CODE REVIEW

File: src/main/resources/assets/botania/lang/zh_CN.lang [hunk 1]
```diff
@@ -3030,7 +3036,7 @@ botania.page.clip1=添加和取出&1透镜&0就像之前做 的那样,要注意
 botania.page.clip2=脉冲枪扩展
 
 # -- ROD OF THE DEPTHS
-botania.entry.cobbleRod=深渊法杖
+botania.entry.cobbleRod=深渊法杖深渊法杖
```
Comment (williewillus) [inline src/main/resources/assets/botania/lang/zh_CN.lang]: dupe?

**Comment by TartaricAcid** [comment]
yes

**Comment by 3TUSK** [comment]
... Atom was lagging my laptop. My bad. Will fix (time zone is eastern standard time)

**Comment by liach** [comment]
Can anyone merge this?

**Comment by williewillus** [comment]
@liach given the original message still says "don't merge yet", no :P

**Comment by liach** [comment]
Where does it say "don't merge yet"? I cannot see. Give me a link.

**Comment by liach** [comment]
Then @3TUSK

**Comment by 3TUSK** [comment]
Now it's my school's spring break, so I could come back from homework+tc6 i18n+bloodmagic i18n+cofh series i18n+...

Question before I start to deal with merge conflict. @williewillus How should I translate "FE" as in `net.minecraftforge.energy`? For me that's impossible because `Forge` is a proper noun.

> I would rather call it `net.minecraftforge.energy` but that's not practical.

**Comment by 3TUSK** [comment]
Notes:
* I put [“能量”](https://en.wiktionary.org/wiki/%E8%83%BD%E9%87%8F) for `net.minecraftforge.energy`. There is really no other way to workaround on this... 
* Include a typo correction in `en_us.lang`.

File: src/main/resources/assets/botania/lang/zh_cn.lang [hunk 2]
```diff
@@ -3886,4 +3886,4 @@ botania.subtitle.terraPickMode=泰拉粉碎者升级
 botania.subtitle.terrasteelCraft=泰拉钢制作完成
 botania.subtitle.thermalily=炽玫瑰消耗熔岩
 botania.subtitle.unholyCloak=罪恶斗篷 激活
-# No translation # botania.subtitle.way=
+# No translation, and please, any attempt of translating this entry will be considered as vandalism to zh_cn.lang # botania.subtitle.way=
```
Comment (liach) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: Why?

File: src/main/resources/assets/botania/lang/zh_cn.lang [hunk 3]
```diff
@@ -2129,14 +2129,14 @@ botania.page.distributor1=合成分配器
...
 # No translation # botania.page.manaVoid1=Even Chad from accounting?
 
 # -- MANA FLUXFIELD
 botania.entry.rfGenerator=魔力转换器
-botania.tagline.rfGenerator=魔力变RF啦!
-botania.page.rfGenerator0=将&4魔力&0引入红石能量场,就可 以将其从自然能量源形态转 为&4红石流量&0.<br>将一魔力发射器对向此方块, 就可把进入其中的&4魔力&0转化 为&4RF&0.之后就能用各种传输&4RF&0 的装置来输出.
-# No translation # botania.page.rfGenerator1=The bullet guys. I bit it.
+botania.tagline.rfGenerator=将 Mana 转化为能量
+botania.page.rfGenerator0=将&4魔力&0引入红石能量场,就可 以将其从自然能量源形态转 为&4能量&0。<br>只需将魔力发射器对准这转 换器，转换器就可以把输入的&4魔力&0转化 为&4能量&0，并输出到邻近的能量传输设备中。
+botania.page.rfGenerator1=传统的挑战者
```
Comment (liach) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: ?

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 2 above)
Comment (3TUSK) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: Prevent *things* from happening

**Comment by liach** [comment]
I suggest you name forge energy 'FE' as modders call it.

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 3 above)
Comment (3TUSK) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: Read the original text.

**Comment by 3TUSK** [comment]
No. There is no official name on that.

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 3 above)
Comment (liach) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: "Keep me out of energy war please" :laughing:

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 2 above)
Comment (liach) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: So this make it display nothing, right? The en file shows some funny string.

Comment (3TUSK) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: Adding `#` at the line-beginning makes it a comment, and thus fallback to `en_us.lang`.

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 3 above)
Comment (3TUSK) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: I wanted to put *Goliath* on my first glance but dropped that idea later. Id est, how about just delete this block? 

> Also, [do quotation properly][citation-1] please

[citation-1]: https://github.com/CovertJaguar/Railcraft/issues/1097#issuecomment-284556917

File: src/main/resources/assets/botania/lang/zh_cn.lang (hunk 2 above)
Comment (liach) [inline src/main/resources/assets/botania/lang/zh_cn.lang]: Thought `##` works like `/**/`

**Comment by 3TUSK** [comment]
@williewillus ping

**Comment by williewillus** [comment]
Sorry for the delay. This is ready now right?

**Comment by 3TUSK** [comment]
No. I am fixing things (shulk me not, deprecated docs, etc) right now, like, literally right now

**Comment by 3TUSK** [comment]
Discussion of 16 hours did not give me an appropriate translation of "Shulk me not". Kept as it is. This PR is ready to go again.

**Pull request merged**